backends/ida/graffiti.py

```diff
@@ -370,6 +306,100 @@ def renamed(self, ea, new_name, is_local):
             }
             lengthy_send(sock, to_bytes(json.dumps(payload)))
 
-hooks = [GraffitiUIHooks(), GrafitiIDBHooks()]
-for hook in hooks:
-    hook.hook()
+
+class GraffitiPlugin(idaapi.plugin_t, idaapi.UI_Hooks):
+    flags = idaapi.PLUGIN_MOD | idaapi.PLUGIN_HIDE
+    comment = "Add support for CPP navigating on top of ida_kernelcache"
+    help = ""
+    wanted_name = "Graffiti"
+    wanted_hotkey = ""
+
+    def __init__(self):
+        super().__init__()
```

Comment:
idapython2 compatibility:

```suggestion
        super(idaapi.plugin_t, self).__init__()
```